### ISSUE TITLE
Replacement of CP72  4.5.0 on pypi

### ISSUE THREAD

**Issue by hartror** [zopefoundation/zope.interface#125]
The SHAs on `zope.interface-4.5.0-cp27-cp27mu-manylinux1_x86_64.whl` changed last week

**Comment by hartror** [comment]
Ah thanks, so previously on linux pip would have been downloading the package via `zope.interface-4.5.0.tar.gz`?

**Comment by mgedmin** [comment]
> Ah thanks, so previously on linux pip would have been downloading the package via zope.interface-4.5.0.tar.gz?

Yes.  The missing wheels were an error that was noticed only recently: https://github.com/zopefoundation/zope.interface/issues/124#issuecomment-428240851
